packages/types/src/pagination.ts

```diff
@@ -5,7 +5,7 @@ import { Client } from "./client";
  *
  * Expected type definition of a paginator.
  */
-export type Paginator<T> = AsyncGenerator<T, T, unknown>;
+export type Paginator<T> = AsyncGenerator<T, undefined, unknown>;
```

Comment:
this appears to be a breaking change, but this is a fix that corrects the type declaration